### PR TITLE
Swipe: Add option to perform haptic feedback after threshold passed

### PR DESCRIPTION
**Pull request by ivaniskandar**
Signed-off-by: Ivan Iskandar <iiiiskandar14@gmail.com>

### Guidelines 

1. You must run the `spotlessApply` task before commiting, either through Android Studio or with `./gradlew spotlessApply`.
2. A PR should be focused and contained. If you are changing multiple unrelated things, they should be in separate PRs.
3. A PR should fix a bug or solve a problem - something that only you would use is not necessarily something that should be published.
4. Give your PR a detailed title and description - look over your code one last time before actually creating the PR. Give it a self-review.

**If you do not follow the guidelines, your PR will be rejected.**

### CODE REVIEW

File: sample/src/main/java/com/afollestad/recyclicalsample/MainActivity.kt [hunk 1]
```diff
@@ -79,6 +79,7 @@ class MainActivity : AppCompatActivity() {
           toast("Delete: $item")
           true
         }
+        hapticFeedbackEnabled = true
```
Comment (afollestad) [inline sample/src/main/java/com/afollestad/recyclicalsample/MainActivity.kt]: I'd prefer having this as a method call, like 
```kotlin
... {
   hapticFeedbackEnabled()
}
```

File: swipe/src/main/java/com/afollestad/recyclical/swipe/SwipeAction.kt [hunk 2]
```diff
@@ -59,6 +59,9 @@ class SwipeAction<IT : Any>(private val context: Context) {
   internal var textPaint: TextPaint? = null
   private var textBounds: Rect? = null
 
+  /** To perform a haptic feedback when swipe threshold is passed. */
+  var hapticFeedbackEnabled = false
```
Comment (afollestad) [inline swipe/src/main/java/com/afollestad/recyclical/swipe/SwipeAction.kt]: So, this should be private and there should be a method that sets it to true 🙂

**Review by afollestad — CHANGES_REQUESTED** [review]

**Comment by ivaniskandar** [comment]
So @afollestad, how is it?

**Comment by afollestad** [comment]
Can you rebase with this repo's master branch? I fixed the Travis CI build

**Comment by ivaniskandar** [comment]
Done, now the checks is all green

**Review by afollestad — APPROVED** [review]
Thanks 👍

**Pull request merged**